src/JSTest/Settings/JSTestSettings.cs

```diff
@@ -68,6 +68,11 @@ public string JavaScriptTestFrameworkAsString
         [DataMember]
         public string TestFrameworkConfigJson { get; set; }
 
+        [DataMember]
+        public string AttachmentsFolder { get; set; }
+
+        [DataMember]
+        public bool UploadAttachments { get; set; }

```

Comment:
i think this should be uploaded in any case. hence we dont need this flag

Reply:
Fixed.